tests/provisioning/provision_test.go

```diff
@@ -29,7 +29,7 @@ type ProvisionTestSuite struct {
 	terraformOptions   *terraform.Options
 }
 
-func (p *ProvisionTestSuite) SetupSuite() {
+func (p *TFPProvisionTestSuite) SetupSuite() {
```

Comment:
supernit; I always feel that abbreviation of all caps next to another word is confusing

Reply:
Resolved